frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,104 @@
+Feature: Image Vulnerability for each Project
+    As a user I would like to see Image Vulnerability for each Project
+
+Background:
+    Given user is in Developer perspecitve
+    And user in project namespace
+    And quay container security operator installed
+
+
+@regression
+Scenario: Image Vulnerability pop over in project overview    
+   Given user is in topology of a project
+   When user goes to project tab on sidebar
+   And user sees Status section in overview tab
+   And user sees vulnerabilities with number mentioned in Status section
+   And user clicks on the Image Vulnerabilities
+   Then user sees Image Vulnerability breakdown pop over
+   And user sees Vulnerable Container Images and Fixable Container Images
+   And user closes the pop over 
```

Comment:
This is the actions and it shouldn't be in the Then statement

Reply:
Edited